db/migrate/20220113134436_create_join_table_pokemons_abilities.rb

```diff
@@ -2,7 +2,7 @@ class CreateJoinTablePokemonsAbilities < ActiveRecord::Migration[6.1]
   def change
     create_join_table :pokemons, :abilities do |t|
       t.index [:pokemon_id, :ability_id]
-      t.index [:ability_id, :pokemon_id]
+      # t.index [:ability_id, :pokemon_id]
```

Comment:
Recomiendo mucho eliminar la linea en lugar de comentarla (más si se hace commit del código), pero no afecta para nada el funcionamiento, así que no hay problema si se conserva así.

Reply:
Will fix! Thanks for the feedback! ❤️